HelpSource/scdoc.css

```diff
@@ -633,17 +649,17 @@ div#search_box {
 }
 
 #search_input {
-    border: 1px solid #aaa;
-    background: #eee;
+    border: 1px solid var(--sc-neutral);
+    background: var(--sc-bg-color);
```

Comment:
Now this background should either be `var(--color-input-bg)`, or removed (to inherit the same value)